apps/aecore/src/aec_chain.erl

```diff
@@ -112,15 +112,17 @@ start_link(GenesisBlock) ->
 stop() ->
     gen_server:stop(?SERVER).
 
-%% Returns the highest known block in the chain with its state trees.
+%% Returns the highest known block in the chain
```

Comment:
Reading the function body, I understand this "with its state trees." can be reinstated.